content/en/synthetics/_index.md

```diff
@@ -2,79 +2,72 @@
+{{< img src="synthetics/api_test.png" alt="API tests"  style="width:100%;">}}
 
-### Write a query
+### Record Browser tests
 
-To search for specific text across all checks, titles, and notification messages, enter your text in the search bar.
+Record end-to-end tests to monitor how your customers experience your webpages from around the world using [Synthetics Browser tests][2].
 
-Otherwise, use Boolean operators (`AND`, `OR`, and `NOT`) and parentheses to write complex queries using any check field:
+{{< img src="synthetics/browser_test.gif" alt="Browser tests"  style="width:100%;">}}
 
-* Regular expressions are not supported
-* Single-character wildcard (`?`) is not supported, but the general wildcard (`*`) is
-* Proximity searches are not supported, but the [fuzzy][1] operator is
-* Ranges are not supported
-* Boosting is not supported
+### Launch Private Locations
 
-Finally, the following characters are reserved: `-`, `(`, `)`, `"`, `~`, `*`, `:`, `.`, and whitespace. To search check fields that include any of them, wrap the field string in quotes: `status:("OK") AND "doc-check"` is a valid query string; `status:("OK") AND doc check` is not.
+Use [Synthetics Private Locations][3] to monitor internal APIs and websites or to create custom locations in areas that are mission-critical to your business.
 
-## Create a check
+{{< img src="synthetics/private_locations.png" alt="Private locations"  style="width:100%;">}}
 
-Select **Create a New check +** in the upper right corner of the Synthetics page to create an [API tests][2] or a [browser test][3]. When a test fails, the endpoint is not retested: the uptime directly considers the endpoint as `down`.
+### Run tests from your CI
 
-{{< img src="synthetics/create_a_check.png" alt="Create a check"  style="width:80%;">}}
+ Include your [Synthetic tests in your CI pipelines][4] and start shipping without fearing faulty code might make it to production.
 
-## Metrics
+ {{< img src="synthetics/ci.png" alt="CI tests"  style="width:100%;">}}
 
-The following metrics are generated by Synthetics checks:
+### Connect Synthetics and Traces
 
-* Metrics starting with `synthetics.browser.*` come from your [browser tests][3].
-* Metrics starting with `synthetics.http.*` come from your [API HTTP tests][4].
-* Metrics starting with `synthetics.ssl.*` come from your [API SSL tests][5].
+Use the [out of the box integration between Synthetic tests and APM traces][5] to find the root cause of failures across frontend, network and backend requests.
 
-{{< get-metrics-from-git "synthetics" >}}
+{{< img src="synthetics/synthetics_traces.gif" alt="Synthetics" style="width:100%;">}}
 
 ## Further Reading
```

Comment:
After GJ review, could we do:
1. Introducing Datadog Synthetics
2. User experience monitoring with Datadog Browser Tests
3. Best practices for creating end-to-end tests
4. Guides
5. Troubleshooting Synthetics

The Settings page is quite empty at the moment so we'd rather not link to it.